scripts/build-images-2019.ps1

```diff
@@ -0,0 +1,30 @@
+# Show statements as they run.
+Set-PSDebug -Trace 2
+# Exit on error.
+$ErrorActionPreference = "Stop"
+
+Write-Output '-- host info ---'
+Write-Output $PSVersionTable
+Write-Output (Get-WMIObject win32_operatingsystem).name
+Write-Output (Get-WMIObject win32_operatingsystem).OSArchitecture
+
+# The servercore images must be built before the nanoserver images.
+cd 2.1.0/servercore1809
+Write-Host "building servercore1809"
+docker build --tag nats:2.1.0-servercore1809 .
+cd ../nanoserver1809
+Write-Host "building nanoserver1809"
+docker build --tag nats:2.1.0-nanoserver1809 .
+
+# We can't test these images on GitHub CI. We get the following error on both
```

Comment:
Still true? You have made some changes in the official images PR by adding a constraint on servercore.

Reply:
Yeah, I think so. The constraint stuff is only for Docker's CI. I think the problem is that our GitHub Actions CI doesn't have the right version of Windows available. We only get windows-2016 or windows-2019, but I think we need something like windows-1803.

Not entirely sure, but it seems like it based on this.
https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility